app/controllers/verify_controller.rb

```diff
@@ -3,5 +3,19 @@ class VerifyController < ApplicationController
 
   render_if -> { IdentityConfig.store.idv_api_enabled }, only: [:show]
 
-  def show; end
+  def show
+    @app_data = app_data
+  end
+
+  private
+
+  def app_data
+    session[:idv_api_store_key] ||= Base64.strict_encode64(OpenSSL::Random.random_bytes(32))
```

Comment:
To avoid magic number and make it clearer what's happening, consider instead something equivalent to:

```rb
OpenSSL::Cipher.new('aes-256-gcm').encrypt.random_key
```

Reply:
Should it be `user_session`?

Reply:
> Should it be `user_session`?

It could be, but for the purpose of demonstration, this uses the routes from #6177, which aren't authenticated, so you can visit this page while being logged out, and `user_session` would not apply. As mentioned in the original PR description, I expect a final implementation would use `document_capture_sessions` instead, to persist and associate the key/IV with the user.

Reply:
Ahhh, got it, thanks!